src/Pyz/Yves/ContactUs/Controller/IndexController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Pyz\Yves\ContactUs\Controller;
+
+use Generated\Shared\Transfer\ContactUsTransfer;
+use Spryker\Yves\Kernel\Controller\AbstractController;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \Pyz\Client\ContactUs\ContactUsClientInterface getClient()
+ */
+class IndexController extends AbstractController
+{
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Spryker\Yves\Kernel\View\View|\Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function indexAction(Request $request)
+    {
+        $contactUsForm = $this
+            ->getFactory()
+            ->createContactUsForm()
+            ->handleRequest($request);
+
+        if ($contactUsForm->isSubmitted() && $contactUsForm->isValid()) {
+
+            $formData = $contactUsForm->getData();
```

Comment:
To avoid this routine mapping, you can use the form option `data_class` (see https://symfony.com/doc/current/forms.html#creating-form-classes)

Reply:
also, our Transfer objects supports mapping from array, check `fromArray()` method of `AbstractTransfer`.